timemachine/fe/rbfe.py

```diff
@@ -174,8 +188,8 @@ def setup_initial_states(
     st: SingleTopology
         A single topology object
 
-    host_config: HostConfig or None
-        Configurations of the host. If None, then a vacuum state will be setup.
```

Comment:
nit: Use `setup_optimized_host` to generate the host.

Reply:
Clarified in https://github.com/proteneer/timemachine/pull/966/commits/2ea7cabb92785738d8e6199b94d1f8545f271a5e